app/bundles/CoreBundle/Controller/AbstractStandardFormController.php

```diff
@@ -727,14 +730,15 @@ protected function getTemplateName($file, string $engine = self::ENGINE_TWIG)
         $originalFile = $file;
         if (self::ENGINE_TWIG === $engine && strpos($file, '.php')) {
             $file = str_replace('.php', '.twig', $file);
+            trigger_deprecation('mautic', '5.x', 'Using PHP Templates is deprecated, use Twig Templates instead.');
```

Comment:
https://github.com/symfony/deprecation-contracts

I would like to keep this here in order to make sure that anything else using PHP templates will trigger a deprecation error to show up in the logs.

Reply:
Good idea!